app/src/main/res/values/strings.xml

```diff
@@ -301,4 +301,6 @@
     <string name="app_shortcut_last_added_long">@string/last_added</string>
     <string name="app_shortcut_last_added_short">@string/last_added</string>
     <string name="playlist_is_empty">Playlist is empty</string>
+    <string name="snack_bar_action_undo">UNDO</string>
+    <string name="snack_bar_title_removed_song">\u0020Removed</string>
```

Comment:
Why the unicode space?

Reply:
Since there is always going to be a space after the song I decided to hard code it. It will always be "SONG" + " Removed". If you want I can add the space to the "soft" code.

Reply:
Ideally the string should be a formattable string like `%s removed`, and you'd just use `String#format` in the code.